### ISSUE TITLE
No Error: 'Configuration file not found. Using default'

### ISSUE THREAD

**Issue by janitor-of-code** [zavoloklom/docker-compose-linter#131]
## Version

main, 2.2.2

## YAML file where the error occurs

all

## Command used to run

`docker run ... zavoloklom/dclint -r --config wrong/path/.dclintrc`

## Current behavior (console output)

> make: *** [Makefile:32: compose-lint] Error 1

## Expected behavior

> `Warning: Configuration file not found. Using default`

## Context

so I was being dumb today, forgot to update a submodule commit and spend a while debugging that.  
Through that I also realised that this tool doesn't complain if the config path is wrong, it just exits.  

I've already looked at the code a bit though I'm not confident enough to make a pull if that's fine.  
It seems it's a problem with the implementation of cosmiconfig:  

In src/config/config.ts:  
```typescript
async function loadConfig(configPath?: string): Promise<Config> {
  const logger = Logger.init();
  const explorer = cosmiconfig('dclint');

  const result = configPath ? await explorer.load(configPath) : await explorer.search();

  if (result && result.config) {
    logger.debug(LOG_SOURCE.CONFIG, `Configuration load from ${result.filepath}.`);
    const config = result.config as unknown as Config;
    return validateConfig(config);
  }
  logger.debug(LOG_SOURCE.CONFIG, 'Configuration file not found. Using default');
  return getDefaultConfig();
}
```

the part `await explorer.load(configPath)` in line 37 actually throws an error [as per the docs](https://github.com/cosmiconfig/cosmiconfig#explorerload) if the file does not exist or cannot be loaded.  
Thus the logger.debug-statement in line 44 (btw I'd appreciate bumping that higher than just 'debug')  is only reached if the file is empty.  
On a malformed or missing file the function throws an error which is caught in lines 93f of src/cli/cli.ts  

```typescript
const config = await loadConfig(cliArguments.config).catch((error) => {
  process.exit(1);
});
```
